@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -10,6 +10,30 @@ import * as THREE from "three";
 
 const epsilon = 0.000001;
 
+/**
+ * Structure that holds near, far planes distances and maximum visibility range.
+ */
+export interface ViewRanges {
+    /**
+     * Distance from camera to near clipping plane along camera eye vector.
+     * @note This value is always positive and in camera space, should be bigger then zero.
+     */
+    near: number;
+    /**
+     * Far clipping plane distance in camera space, along its eye vector.
+     * @note Should be always positive and bigger then [[near]] plane distance.
+     */
+    far: number;
+    /**
+     * Maximum possible visibility range in camera space, regardless of camera orientation.
```

Comment:
I'm still not understanding what maximum means based on this description.

`Holds the maximum value that may be returned for [[far]] plane distance at certain camera position and orientation`

Am I correct in saying that this value acts to `override` the far plane in certain circumstances?

Why does this have to be part of the ClipPlanesEvaluator? To me this is unnecessarily complicating this interface, by forcing users to return some value which they have to experiment with so that they don't break the fog. Couldn't the fog calculation been smart enough to take the tilt into consideration, and thus not complicate the interface?

Or am I missing some detail?